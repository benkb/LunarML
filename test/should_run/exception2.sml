fun main () = ((raise Fail "exception 1")
               handle Fail x => ( print (x ^ "\n")
                                ; raise Fail "exception 2"
                                )
              )
              handle Fail y => print (y ^ "\n");
main ();
