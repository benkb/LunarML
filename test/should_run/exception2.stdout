exception 1
exception 2
